solutions/YS Vertex Set Path Composite.mdx

```diff
@@ -31,12 +31,12 @@ The update queries can be implemented using segment tree point updates on both s
 ```cpp
 
 #include <bits/stdc++.h>
-#define ll long long
+using ll = long long;
+using namespace std;
 #define pi pair<ll,ll>
```

Comment:
not updated

Reply:
can you clarify what I need to update that I have not done ?

Reply:
`using pi = pair<ll,ll>;`